borg/testsuite/helpers.py

```diff
@@ -259,6 +260,46 @@ def testInvalidUnicode(self):
         assert e.match(str(b"ba\x80/foo", 'latin1'))
 
 
+class PatternFromFileTestCase(BaseTestCase):
+
+    files = [
+        '/etc/passwd', '/etc/hosts', '/home',
```

Comment:
maybe don't use filenames that are somehow loaded with meaning if it is not needed for the test purpose.
someone / something might think borg is accessing their passwd file...